src/material-experimental/mdc-checkbox/checkbox.ts

```diff
@@ -388,4 +388,17 @@ export class MatCheckbox
       nativeCheckbox.nativeElement.indeterminate = value;
     }
   }
+
+  /**
+   *  Prevent click events that come from the `<label/>` element from bubbling. This prevents the
+   *  click handler on the host from triggering twice when clicking on the `<label/>` element. After
+   *  the click event on the `<label/>` propagates, the browsers dispatches click on the associated
+   *  `<input/>`. By preventing clicks on the label by bubbling, we ensure only one click event
+   *  bubbles when the label is clicked.
+   **/
+  protected preventBubblingFromLabel(event: MouseEvent) {
```

Comment:
I think that this should be prefixed with an underscore since we don't want people using it.

Reply:
fixed